packages/gestalt/src/Checkbox.jsdom.test.js

```diff
@@ -0,0 +1,23 @@
+// @flow
```

Comment:
Extracted out of `src/Checkbox.test.js` since clicking is a browser only event (doesn't happen on the server)